src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -19,20 +19,20 @@ export function getStaticProps(context) {
   };
 }
 
-This guide will help you migrate v5 Storage implementations to the new functional API's in v6.
+This guide will help you migrate v5 Storage implementations to the new functional APIs in v6.
 
 ## Deprecated Options
 
 <Callout>Many input and output properties have been deprecated due to low usage or added complexity. If you would like to see these options re-implemented in v6, feel free to open a [feature request](https://github.com/aws-amplify/amplify-js/issues/new?assignees=&labels=pending-triage&projects=&template=2.feature_request.yaml) in our github repository.</Callout>
 
-The input options below have been removed in v6 from Storage API's:
+The input options below have been removed in v6 from Storage APIs:
 
-- `provider`: _Custom providers are no longer supported in v6._
+- `provider`: _Custom providers are no longer supported in v6. All examples below use the input/output values expected from the Default AWSS3Provider._
 - `customPrefix`: _You can [add a prefix resolver](/[platform]/build-a-backend/storage/configure-access/#customize-object-key-path) to your Storage Config to customize the key prefix per accessLevel. See the [Using a Custom Prefix](/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/#using-a-custom-prefix) header for migration details._
-- `track`: _Tracking Storage events in Hub is not yet supported in v6._
+- `track`: _Auto-tracking Storage events in pinpoint is Deprecated in V6. You can track Storage events [using the Analytics category](/[platform]/build-a-backend/more-features/analytics/record-events/)._
```

Comment:
Nit: Should we reuse the language "has been removed"? Deprecated usually implies it's still around but no longer supported/recommended

Reply:
Yes, has been removed overall is better.